src/service/owm/weather.cpp

```diff
@@ -149,9 +149,16 @@ void weather_start() {
 
     firstRunSemaphore = xSemaphoreCreateBinary();
 
-    apiKey = prefs_get_string(PREFS_KEY_WEATHER_APIKEY, String(WEATHER_API_KEY));
-    latitude = prefs_get_string(PREFS_KEY_WEATHER_LAT, String(WEATHER_LAT));
-    longitude = prefs_get_string(PREFS_KEY_WEATHER_LON, String(WEATHER_LON));
+    apiKey = prefs_get_string(PREFS_KEY_WEATHER_APIKEY);
```

Comment:
Memo for future context: these defaults are not needed anymore, since they will be copied from the build config inside the [prefs service init routine](https://github.com/vladkorotnev/plasma-clock/blob/main/src/service/prefs.cpp#L40-L45), so this should be safe enough

Reply:
Got it!